reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -619,6 +628,18 @@ class StaticHtmlReporter : Reporter {
     }
 }
 
+private fun EM.insertDetectionSource(provenance: Provenance?) {
```

Comment:
Imperative function names do not match well with the DSL style, what about `provenanceLink()` instead?

Reply:
I don't have a strong opinion here, so, changed.